cove/settings.py

```diff
@@ -86,6 +85,21 @@
                       'record': 'record-package-schema.json'},
         'default': None
     },
+    'schema_version': {  # Default schema version
+        'cove-ocds': '1.0',
+        'cove-360': None,
+        'default': None
+    },
+    'schema_version_choices': {
+        # {version: (display, url)}
+        'cove-ocds': {
```

Comment:
These options need some kind of way to decide the ordering when going in the dropdown list on the frontend.  So an ordered dict makes more sense.

I also do not understand this structure.  It says # version: (display, url) but display is used to go into the database not version.